process/incident_management.md

```diff
@@ -0,0 +1,273 @@
+* Role is handed over from one shift to the other until full resolution
+* Helps the Incident Owner document the RCA and PMR Action Items.
+
+### Supporting SRE / Parallel Investigator
+A parallel investigator is someone who will be involved hands-on in investigating and helping the fast recovery of the incident.
+A parallel investigator will get involved either by the Incident Commander or by any other members of the slack channel.
+
+#### Responsibilities
+The investigators have the responsibility to describe the action they are taking in the dedicated slack channel to ensure that there is awareness on the steps being taken for the recovery of everyone involved in the process. This can also be done by someone else assisting in the investigation while the main investigator is busy with the recovery.
+
+### Incident Commander
+
+The key element for this role is to ensure that the Incident Commander is solely focused on "oiling the machinery" of incident recovery and will not participate in the hands-on resolution. In Fact the main goal for the Incident Commander is to deal with the overall coordination overhead of the incident recovery, with a cool head, while the investigators are focused 100% on performing the recovery. This will allow faster recovery, better communication flows and less errors during the stress of the incident management.
+
+The Incident Commander can elect someone to be the "scribe" during the incident. The duty of the scribe is to report in the RCA document all the important information (technical details, timeline, ...) while the Incident Commander is focused on helping with the coordination.
+
+#### Responsibilities
+The coordination work under the responsibility of the Incident Commander is as follows:
+1. If the incident has been uncovered by SRE and doesn't have an existing "external" ticket (Jira or ICM for ARO) then the Incident Commander will create a Jira ticket to track the overall life of the incident.
+2. Right after the Incident Commander will create a dedicated Slack channel:
+  * Making sure that we have a dedicated IM channel opened specifically for the ongoing incident
+    1. Channel is public
+    2. Naming convention for the IM channel should start with the ticket number and the clusterid if available.
+       * Examples:
+         * #inc1096029-<CLUSTERID>
+         * #OHSS-1234-title
+    3. The description of the IM channel should record:
+      * Incident Commander
+      * Incident Tech Lead
+      * Bridge link
+      * A small description of the issue
+    4. Making sure the right people are invited in the slack channel (including customer support if need be)
+    5. Making sure that all communications and information flows are recorded in the IM channel (it will help with the Post Mortem Review process and making sure that we have the right timeline)
+  3. Using IM, post a message to the relevant channel with:
+    * **link to bridge**
+    * **link to the dedicated slack channel (see point 2.)**
```

Comment:
remove ref of Slack and just mention IM channel

Reply:
updated in [f799548](https://github.com/operate-first/sre/pull/17/commits/f79954870cd4360b9ede03ca11adbd53bd87232d)